src/main/java/DateTime.java

```diff
@@ -0,0 +1,121 @@
+public class DateTime {
+
+    String dateTime;
+    String day;
+    String month;
+    String year;
+    String time;
+
+    public DateTime(String dateTime) {
+        this.dateTime = dateTime;
+    }
+
+    public void processDate(String d) {
+        String[] splitDate = d.split("/");
+        processDay(splitDate[0]);
+        processMonth(splitDate[1]);
+        year = splitDate[2];
+    }
+
+    public void processDay(String d) {
+        int dayNum = Integer.parseInt(d);
+        if (dayNum % 10 == 1) {
+            day = d + "st";
+        } else if (dayNum % 10 == 2) {
+            day = d + "nd";
+        } else if (dayNum % 10 == 3) {
+            day = d + "rd";
+        } else {
+            day = d + "th";
+        }
+    }
+
+    public void processMonth(String m) {
+        switch (Integer.parseInt(m)) {
+        case 1:
+            month = "January";
+            break;
+        case 2:
+            month = "February";
+            break;
+        case 3:
+            month = "March";
+            break;
+        case 4:
+            month = "April";
+            break;
+        case 5:
+            month = "May";
+            break;
+        case 6:
+            month = "June";
+            break;
+        case 7:
+            month = "July";
+            break;
+        case 8:
+            month = "August";
+            break;
+        case 9:
+            month = "September";
+            break;
+        case 10:
+            month = "October";
+            break;
+        case 11:
+            month = "November";
+            break;
+        case 12:
+            month = "December";
+            break;
+        }
+    }
+
+    public void processTime(String t) {
```

Comment:
The amount of effort put in here should be applauded but there is a simpler way in the one that I did, in that you can use the dateFormat class and the methods within and it makes all these date conversions quite automatic

Reply:
yeah i agree should've search for a class that would do this automatically